doc/095_considerations.rst

```diff
@@ -0,0 +1,36 @@
+..
+  Normally, there are no heading levels assigned to certain characters as the structure is
+  determined from the succession of headings. However, this convention is used in Python’s
+  Style Guide for documenting which you may follow:
+
+  # with overline, for parts
+  * for chapters
+  = for sections
+  - for subsections
+  ^ for subsubsections
+  " for paragraphs
+
+######################
+Special Considerations
+######################
+
+************************************
+Large Repositories and Min Pack Size
+************************************
+
+In certain instances, such as very large repositories, it is desired to have larger pack
+sizes to reduce the number of files in the repository.  Notable examples are OpenStack
+Swift and some Google Drive Team accounts, where there are hard limits on the total
+number of files.  This can be achieved by either using thei ``--min-packsize`` flag
+or defining the ``$RESTIC_MIN_PACKSIZE`` environment variable.
+
+The side effect of increasing the pack size is increased client memory usage.  A bit of
```

Comment:
Maybe add an explanation that restic uses PackSize * ParallelWriters amount of  memory for storing packs, where ParallelWriters (technically: `archiver.Options.SaveBlobConcurrency`) defaults to the number of CPUs used.

Reply:
Maybe you could also consider adding a flag to specify `archiver.Options.SaveBlobConcurrency`. This would allow to use a larger --min-packsize on multi-core machines with low memory.

Reply:
I like that idea.  I'll implement that.